v2/callctx/callctx.go

```diff
@@ -38,6 +38,14 @@ import (
 )
 
 const (
+	// XGoogFieldMask is the canonical header key for the [System Parameter]
+	// that specifies the response read mask. The value(s) for this header
+	// must adhere to format described in [fieldmaskpb].
+	//
+	// [System Parameter]: https://cloud.google.com/apis/docs/system-parameters
+	// [fieldmaskpb]: https://google.golang.org/protobuf/types/known/fieldmaskpb
+	XGoogFieldMask = "x-goog-fieldmask"
```

Comment:
nit: should this have the word Header or Key in the name? I am just thinking if we add more of these in the future it would be nice to have a grouping of them

Reply:
Sure thing, added the `Header` suffix to the name.